README.md

```diff
@@ -1,35 +1,58 @@
 # table_plane_extractor
 
-Service for horizontal table plane extraction
+- Service for horizontal table plane extraction.  
+- Service for getting objects (or multi-objects blobs if they are too close to each other) that are placed on the table plane.
 
 ## Dependencies ##
 - [ROS Noetic](http://wiki.ros.org/noetic/Installation/Ubuntu) 
-- [Open3d](http://www.open3d.org/docs/release/)
-- [open3d_ros_helper](https://github.com/SeungBack/open3d-ros-helper) 
-- [tf2_ros](http://wiki.ros.org/tf2_ros)
+- [requirements.txt](requirements.txt)
+- [CMakeLists.txt](CMakeLists.txt)
 
-## Service
+## Services
+
+### TablePlaneExtractor
+Table plane extractor who takes the point cloud as input and return possible horizontal planes with plane equation ( $x, y, z, d \rightarrow a x + b y + c z + d = 0$ ) and inlier cloud.
+
+**Input:** sensor_msgs/PointCloud2 pointcloud  
+**Output:** table_plane_extractor/Plane[] planes, sensor_msgs/PointCloud2[] clouds
 
-Table plane extractor who takes the point cloud topic as input and return possible horizontal planes with plane equation (x, y, z, d -> a * x + b * y + c * z + d = 0) and inlier cloud.
-Input: string pointcloud_topic
-Output: table_plane_extractor/Plane[] planes, sensor_msgs/PointCloud2[] clouds
 ```
 src/table_plane_extractor_srv.py
 ```
 
-## Demo
-
-You can find a demo code in the File 
+### GetObjectsOnTable
+Service that returns bounding boxes of objects found on a table plane.  
+**Input:** sensor_msgs/PointCloud2 scene_pointcloud  
+**Output:** vision_msgs/BoundingBox3DArray detected_objects
 ```
-src/test_plane.py
+src/get_objects_on_table.py
 ```
 
 ## Startup
 
-You can start the service with
+You can start the TablePlaneExtractor service with
 ```
 roslaunch table_plane_extractor table_plane_extractor.launch
 ```
+You can start the GetObjectsOnTable service with
+```
+roslaunch table_plane_extractor get_objects_on_table.launch
+```
+
+## Demo
```

Comment:
Nice presentation overall, should Demo be switched out for Test?

Reply:
Originally the files were intended to show example usages (hence demo), but I guess you are right, they will mostly be used for testing the services, so I updated the README.